cdap-ui/app/tracker/services/my-tracker-api.js

```diff
@@ -16,7 +16,7 @@
 
 function myTrackerApi(myCdapUrl, $resource, myAuth, myHelpers, CDAP_UI_CONFIG) {
   var url = myCdapUrl.constructUrl,
-      searchPath = '/namespaces/:namespace/metadata/search?target=stream&target=dataset&target=view',
+      searchPath = '/namespaces/:namespace/metadata/search?limit=25&offset=0&target=dataset&target=stream',
```

Comment:
As provided on https://issues.cask.co/browse/TRACKER-177, its not in params list
Could you please confirm